internal/cobraprompt/prompt.go

```diff
@@ -49,9 +51,39 @@ type CobraPrompt struct {
 	OnErrorFunc func(err error)
 }
 
+var ExitError = errors.New("exit prompt")
```

Comment:
How about `ErrExit` to make it more idiomatic?

Reply:
Sure! https://github.com/hazelcast/hazelcast-commandline-client/pull/35/commits/bdd565a3b18e25cbb9e967db21a395523885a7e1